docs/fedora-messaging.rst

```diff
@@ -91,8 +102,95 @@ configuration file and no options on the command line.
 ``--exchange``
 
     The name of the exchange to bind the queue to. Can contain ASCII letters,
-    digits, hyphen, underscore, period, or colon. If one is not specified, the
-    default is the ``amq.topic`` exchange.
+    digits, hyphen, underscore, period, or colon.
+
+    Setting this option is equivalent to setting the ``exchange`` setting
+    in *all* ``bindings`` entries in the configuration file.
+
+
+publish
+-------
+
+The publish command expects the message or messages provided in ``FILE`` to be
+JSON objects with each message separated by a newline character. The JSON
+object is described by the following JSON schema::
+
+  {
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "description": "Schema for the JSON object used to represent messages in a file",
+    "type": "object",
+    "properties": {
+        "topic": {"type": "string", "description": "The message topic"},
+        "headers": {
+            "type": "object",
+            "properties": Message.headers_schema["properties"],
+            "description": "The message headers",
+        },
+        "id": {"type": "string", "description": "The message's UUID."},
+        "body": {"type": "object", "description": "The message body."},
+        "queue": {
+            "type": "string",
+            "description": "The queue the message arrived on, if any.",
+        },
+    },
+    "required": ["topic", "headers", "id", "body", "queue"],
+  }
```

Comment:
this is great! I think it would be even better if we had a real example below (i.e. we pass in a real world example message into the `fedora-messaging publish -`

Reply:
I went back and look at [my comment from yesterday](https://github.com/fedora-infra/fedora-messaging/pull/43#issuecomment-506059053).. we could use that example if we wanted (or tweak it however you'd like)